SDDC_FX3/USBhandler.c

```diff
@@ -120,11 +123,12 @@ CyFxSlFifoApplnUSBSetupCB (
         }
     } else if (bType == CY_U3P_USB_VENDOR_RQT) {
 
-    	/*
-   	    uint8_t * pd = (uint8_t *) &glEp0Buffer[0];
-    	uint32_t event =( (VENDOR_RQT<<24) | ( bRequest<<16) | (pd[0]<<8) | pd[1] );
-    	CyU3PQueueSend(&EventAvailable, &event, CYU3P_NO_WAIT);
-  */
+		uint8_t * pd = (uint8_t *) &glEp0Buffer[0];
```

Comment:
I suggest add another function instead of MsgParsing. looks like MsgParsing was good for parsing the Events. for vendor command, we can just use another function to avoid encode/decode complex.

Reply:
Thanks! I'll try